python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -362,6 +376,8 @@ cpdef read_parquet(filepaths_or_buffers, columns=None, row_groups=None,
         filters,
         convert_strings_to_categories = False,
         use_pandas_metadata = use_pandas_metadata,
+        skip_rows = skip_rows,
+        num_rows = nrows,
```

Comment:
Let's be consistent with either `num_rows` or `nrows` across the files. @galipremsagar I can't find the same option in `pyarrow.read_table` or `pd.read_parquet` so am not sure which one to pick. If arbitrary, my vote would be `num_rows` to be consistent with C++ counterpart but not a blocker.

Reply:
Yeah not sure which is better.

nrows would be consistent with read_csv, and num_rows would be consistent with libcudf.

Reply:
Let's go with `nrows` then and further the PR to merge!